mesonbuild/backend/vs2010backend.py

```diff
@@ -467,7 +467,7 @@ def generate_projects(self):
 
         # Put the startup project first in the project list
         if startup_idx:
-            projlist = [projlist[startup_idx]] + projlist[0:startup_idx] + projlist[startup_idx + 1:-1]
+            projlist = [projlist[startup_idx]] + projlist[0:startup_idx] + projlist[startup_idx + 1:]
```

Comment:
This is effectively performing the following right:
```python
first = projectlist.pop(startup_idx)
protlist.insert(0, first)
```
right? That seems so much clearer to me, and less likely to reintroduce the same problems with slicing.

Reply:
Sure, done. I was just going for the minimal change and thought 2 chars was tough to beat :)

Reply:
thanks, looks good